File: 프로그래머스/LV0/코딩테스트 입문/k의 개수/k의 개수.py
def solution(i, j, k):
    kc = str(k)
    return sum([ str(i).count(kc) for i in range(i,j+1)])

# def solution(i, j, k):
#     answer = 0
#     kc = str(k)
#     for n in range(i, j+1):
#         for c in str(n):
#             if c == kc:
#                 answer += 1
#     return answer

File: 프로그래머스/LV0/코딩테스트 입문/k의 개수/k의 개수.c
#include <stdio.h>
#include <stdbool.h>
#include <stdlib.h>

int solution(int i, int j, int k)
{
    int answer = 0;
    int cur_n = 0;

    for (int n = i; n <= j; n++)
    {
        cur_n = n;
        while (cur_n)
        {
            if (cur_n % 10 == k)
                answer++;
            cur_n /= 10;
        }
    }
    return answer;
}